core/isolate.rs

```diff
@@ -311,7 +323,7 @@ impl Isolate {
     &mut self,
     js_filename: &str,
     js_source: &str,
-  ) -> Result<(), JSError> {
+  ) -> Result<(), ErrBox> {
```

Comment:
Please add a triple slash comment saying the ErrBox should always be downcast to CoreJsError.